application/src/test/resources/speedtest/travelSearch-expected-results-srr.csv

```diff
@@ -1,6 +1,6 @@
 tcId,nTransfers,duration,cost,walkDistance,startTime,endTime,agencies,modes,routes,stops,details
-1,0,56m36s,0,1458,12:03:49,13:00:25,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:11:20 12:49:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m6s
-1,0,56m36s,0,1458,12:18:49,13:15:25,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:26:20 13:04:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m6s
-1,0,56m36s,0,1458,12:33:49,13:30:25,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:41:20 13:19:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m6s
-2,0,1h3m59s,0,1902,12:48:49,13:52:48,TriMet,BUS,8,prt:6789|prt:5028,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:56:20 13:36 ~ SW Sam Jackson Pk & OHSU(5028) ~ Walk 16m48s
-2,1,1h2m1s,0,1488,12:50:47,13:52:48,TriMet,BUS,73|8,prt:7106|prt:2592|prt:5028,Walk 2m19s ~ NE 21st & Clackamas(7106) ~ BUS 73 12:53:06 13:01 ~ Rose Quarter Transit Center(2592) ~ BUS 8 13:04 13:36 ~ SW Sam Jackson Pk & OHSU(5028) ~ Walk 16m48s
+1,0,56m33s,0,1458,12:03:49,13:00:22,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:11:20 12:49:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m3s
+1,0,56m33s,0,1458,12:18:49,13:15:22,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:26:20 13:04:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m3s
+1,0,56m33s,0,1458,12:33:49,13:30:22,TriMet,BUS,8,prt:6789|prt:6548,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:41:20 13:19:19 ~ SW 11th & Gibbs(6548) ~ Walk 11m3s
+2,0,1h3m56s,0,1902,12:48:49,13:52:45,TriMet,BUS,8,prt:6789|prt:5028,Walk 7m31s ~ NE 15th & Halsey(6789) ~ BUS 8 12:56:20 13:36 ~ SW Sam Jackson Pk & OHSU(5028) ~ Walk 16m45s
+2,1,1h1m58s,0,1488,12:50:47,13:52:45,TriMet,BUS,73|8,prt:7106|prt:2592|prt:5028,Walk 2m19s ~ NE 21st & Clackamas(7106) ~ BUS 73 12:53:06 13:01 ~ Rose Quarter Transit Center(2592) ~ BUS 8 13:04 13:36 ~ SW Sam Jackson Pk & OHSU(5028) ~ Walk 16m45s
```

Comment:
Do you know why all of these speed tests are 3 seconds quicker? Ideally changing the internal angle representation should not change travel time.

Reply:
It is because of the inAngle / outAngle fix for creating the temporary edge has changed the turn time near the origin / destination.